extensions/objects/templates/image-with-regions-editor.component.ts

```diff
@@ -29,41 +29,56 @@ import { CustomSchema } from 'services/schema-default-value.service';
 import { UtilsService } from 'services/utils.service';
```

Comment:
Not throwing error here at `if` checks in this file because we don't want to stop function execution at places where they are used. Stopping function execution there and then throws errors that are not wanted when cropping regions in an uploaded image leading to a bad user experience. Also, this leads to many failures in e2e.